stock_sale_product_volume/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2024 Akretion (<http://www.akretion.com>).
+# @author Mathieu Delva <mathieu.delva@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    total_volume = fields.Float(compute="_compute_total_volume", string="Volume en m3")
```

Comment:
s/en m3/in cubic meter/


Also eventually you add an _m3 suffix in the field name or manage its uom (see how it's done for volume on picking) so cubic meter might not be hardcoded.

Reply:
in fact I think managing the unit (use the product unit squared? use some kind of per company or per sale team configuration?) would make the module way more usable...  cc @sebastienbeau